src/icons/fileIcons.ts

```diff
@@ -1850,5 +1850,23 @@ export const fileIcons: FileIcons = {
         'steadybit.yaml',
       ],
     },
+    {
+      name: 'systemd',
+      fileExtensions: [
```

Comment:
Thank you for your contribution! However I see one issue here. Imagine there's another project that contains files having the file name "info.service" but is not related to systemd. In my opinion most of your suggested file extension are too commonly used also outside of the context of systemd. It might be confusing for people to see a systemd icon for some file which is not related to systemd at all. In this case this could be very likely. 

But still we can think about merging the changes of the languageIcon. Then VS Code decides if a file is a systemd file and then shows the systemd icon - which will be correct then.

Reply:
Hi @PKief , I removed icon from fileIcons.

Reply:
thank you :)